config.go

```diff
@@ -713,7 +673,7 @@ func loadConfig() (*config, []string, error) {
 	if !cfg.UseLEHTTPS && (!fileExists(cfg.GUITLSCert) || !fileExists(cfg.GUITLSKey)) {
 		err := genCertPair(cfg.GUITLSCert, cfg.GUITLSKey)
 		if err != nil {
-			str := "%s: unable to generate dcrpool's TLS cert/key: %v"
+			const str = "%s: unable to generate dcrpool's TLS cert/key: %w"
 			err := fmt.Errorf(str, funcName, err)
 			fmt.Fprintln(os.Stderr, err)
 			return nil, nil, err
```

Comment:
This wasnt showing usage previously, now it is.

Reply:
Suppressed.